pkg/config/security/security.go

```diff
@@ -100,6 +105,33 @@ func CheckEmptyValues(key string, values []string) error {
 	return nil
 }
 
+func CheckValidPathTemplate(key string, paths []string) error {
+	for _, path := range paths {
+		// If path does not contain any path templates, skip the check.
+		if !ContainsPathTemplate(path) {
```

Comment:
I think we want to remove this if now, so we would reject something like `/foo/{name}/bar`.

Though we will need to improve the `Contains(*)` check since that is valid for !ContainsPathTemplate.

Reply:
Updated